specification.markdown

```diff
@@ -386,41 +386,60 @@ The path defines the allowed storage locations of the EBML Element within an EBM
 The path attribute is REQUIRED.
 
 ```
-EBMLFullPath          = [EBMLParentPath] EBMLElementPath
-EBMLParentPath        = EBMLFixedParent EBMLLastParent
-EBMLFixedParent       = *(EBMLPathAtom)
-EBMLElementPath       = EBMLPathAtom / EBMLPathAtomRecursive
-EBMLPathAtom          = PathDelimiter EBMLAtomName
-EBMLPathAtomRecursive = "(1*(" EBMLPathAtom "))"
-EBMLLastParent        = EBMLPathAtom / EBMLGlobalParent
-EBMLGlobalParent      = "(" GlobalParentOccurence "\)"
-EBMLAtomName          = 1*(EBMLNameChar)
-EBMLNameChar          = ALPHA / DIGIT / "-" / "."
+EBMLFullPath          = EBMLParentPath EBMLElement
+
+EBMLParentPath        = PathDelimiter [EBMLParents]
+
+EBMLParents           = 0*IntermediatePathAtom EBMLLastParent
+IntermediatePathAtom  = EBMLPathAtom / GlobalPlaceholder
+EBMLLastParent        = EBMLPathAtom / GlobalPlaceholder
+
+EBMLPathAtom          = [IsRecursive] EBMLAtomName PathDelimiter
+EBMLElement           = [IsRecursive] EBMLAtomName
+
 PathDelimiter         = "\"
-GlobalParentOccurence = [PathMinOccurrence] "*" [PathMaxOccurrence]
+IsRecursive           = "+"
+EBMLAtomName          = ALPHA / DIGIT 0*EBMLNameChar
+EBMLNameChar          = ALPHA / DIGIT / "-" / "."
+
+GlobalPlaceholder     = "(" GlobalParentOccurence "\)"
```

Comment:
Since parenthesis are no longer used for recursion, I suggest a cleaner look and use:

`GlobalPlaceholder     = "(" GlobalParentOccurence ")"`

Reply:
I have a slight preference for:

`GlobalPlaceholder = "(" GlobalParentOccurence ")" PathDelimiter`

As in, instead of `\a\(0-1\)global` to have `\a\(0-1)\global`

Reply:
But that's the point. (*\) means there are plenty of abc\def\ghi\ in place of the placeholder. The \ is "repeated" for each occurence. If you leave it out IMO it's less readable.